pjmedia/include/pjmedia/config.h

```diff
@@ -47,14 +47,14 @@
  * Initial memory block for media endpoint.
  */
 #ifndef PJMEDIA_POOL_LEN_ENDPT
-#   define PJMEDIA_POOL_LEN_ENDPT               512
+#   define PJMEDIA_POOL_LEN_ENDPT               8192
 #endif
 
 /**
  * Memory increment for media endpoint.
  */
 #ifndef PJMEDIA_POOL_INC_ENDPT
-#   define PJMEDIA_POOL_INC_ENDPT               512
+#   define PJMEDIA_POOL_INC_ENDPT               4096
```

Comment:
I propose 4000 instead of 4096, just to be on the conservative side, in case the OS adds additional bytes to the block, to make sure the whole things fits in 4KB page. (similarly for the 8192 and 16384, trim them by ~100 bytes maybe)